apps/server/src/modules/board/repo/recursive-save.visitor.ts

```diff
@@ -185,21 +178,27 @@ export class RecursiveSaveVisitor implements BoardCompositeVisitor {
 		this.visitChildren(externalToolElement, boardNode);
 	}
 
-	visitChildren(parent: AnyBoardDo, parentNode: BoardNode) {
+	private visitChildren(parent: AnyBoardDo, parentNode: BoardNode) {
 		parent.children.forEach((child) => {
 			this.registerParentData(parent, child, parentNode);
 			child.accept(this);
 		});
 	}
 
-	registerParentData(parent: AnyBoardDo, child: AnyBoardDo, parentNode: BoardNode) {
+	private registerParentData(parent: AnyBoardDo, child: AnyBoardDo, parentNode: BoardNode) {
 		const position = parent.children.findIndex((obj) => obj.id === child.id);
 		if (position === -1) {
 			throw new Error(`Cannot get child position. Child doesnt belong to parent`);
 		}
 		this.parentsMap.set(child.id, { boardNode: parentNode, position });
 	}
 
+	private saveRecursive(boardNode: BoardNode, anyBoardDo: AnyBoardDo): void {
```

Comment:
I don't think this is helpful.
Replacing 2 line for 1 is not good enough to introduce a redirection.

Reply:
I see that as not clean code.
If you are working on some code base for a while you are more in write mode and DRY will trick you thinking that any small increment in code reduction is good.
It is not => I feel every much our code base is hard to read exactly because we are abstracting out too much and need to visit all indirections.

There is always a balance to make between DRY and LoB.
And a 1 line reduction is not convincing.

Here is an interesting read.
https://htmx.org/essays/locality-of-behaviour/